main_routers/characters_router.py

```
@@ -2320,7 +2332,104 @@ async def voice_clone(file: UploadFile = File(...), prefix: str = Form(...), ref
+                'prefix': prefix,
+                'audio_md5': audio_md5,
+                'ref_language': ref_language,
+                'minimax_language': minimax_lang,
+                'provider': 'minimax',
+                'created_at': datetime.now().isoformat()
+            }
+
+            try:
+                _config_manager.save_voice_for_api_key(minimax_storage_key, voice_id, voice_data)
+                logger.info(f"MiniMax voice_id 已保存到音色库: {voice_id}")
+            except Exception as save_error:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)